inference-engine/src/gna_plugin/gna_model_serial.cpp

```diff
@@ -391,8 +395,15 @@ void GNAModelSerial::Export(void * basePointer, size_t gnaGraphSize, std::ostrea
     // writing memory information
     writeBits(static_cast<uint32_t>(states.size()), os);
     for (auto && state : states) {
-        writeBits(offsetFromBase(state.first), os);
-        writeBits(state.second, os);
+        void* gna_ptr = nullptr;
+        uint32_t reserved_size = 0;
+        std::string name = "noname";
```

Comment:
There is no point in initializing it with "noname" as it will be overwritten in the next step.